### DISCUSSION TITLE
highlightjs 버전이 명시 또는 버전 업

### DISCUSSION THREAD

**Discussion by spearkkk** [pronist/hELLO#68] [Ideas]
**환경 (버그가 발생한 환경을 적어주세요.):**

- 운영체제: [e.g. windows] osx
- 브라우저: [e.g. chrome, safari] chrome
- 버전: [e.g. 22] 
- 스킨 버전 (스킨 버전은 index.xml 에서 확인): [e.g 3.4.0] 3.5.5

**설명**
버그가 어떻게 발생하는지 알려주세요. 대략적인 버그의 설명을 적어주시면 됩니다.
[e.g. 코드 하이라이팅이 동작하지 않습니다.]

가이드라인에서는 코드 블록에 대한 테마를 highlight js에서 찾아 넣으면 된다고 명시되어 있지만,
실제로 현재(2021.12.30.) 기준으로 동작하지 않은 테마가 있습니다.
현재 코드를 추측해보는 결과 테마 명을 설정하면 해당 테마 명으로 cdn에서 가져오는 것으로 보여지는데,
그 링크가 highlight 버전에 따라 다릅니다

현재 hightligt의 최신 버전으로 보면
`https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.3.1/styles/base16/gruvbox-dark-hard.min.css`
이지만,

블로그 테마에 적용된 버전으로는 
`https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.1.2/styles/gruvbox-dark.min.css`
으로 가져와야합니다.

highlight 버전을 명시해주시거나 버전업을 해주실 수 있을까요?

**발생 과정**
버그가 어떤 과정을 통해 발생하는지 구체적으로 적어주세요.
[e.g. 테마 변경 버튼을 통해 화이트 모드로 변경시 코드 하이라이팅의 색상이 변하지 않습니다. ...]

위의 설명

**스크린샷 및 링크**
버그가 발생한 상황을 가능하다면 스크린샷을 첨부해주세요. 또한 버그가 발생한 모습이 있는 포스트의 주소를 첨부해주시는 것도 아주 좋습니다!

지금은 10.1.2 버전에 있는 것을 가져오도록 설정함.

**Comment by pronist** [comment]
@spearkkk 이부분은 조치해야겠네요. 내년에 3.6.0 버전 릴리즈할 때 반영하도록 하겠습니다.

**Comment by pronist** [comment]
@spearkkk 3.6.1 버전에서 문제가 해결되었음을 알립니다. 배포 페이지에도 명시해야겠네요 :)

**Comment by spearkkk** [comment]
@pronist 고려해주셔서 감사합니다.
